### DISCUSSION TITLE
Export data format setting: use the system values

### DISCUSSION THREAD

**Discussion by gaborsodi** [dbeaver/dbeaver#20043] [Q&A]
**Is your feature request related to a problem? Please describe.**
When exporting results, how can I change the delimiter in case of decimal fractions? It is always "." (and the thousand delimiter is always ',') .

![image](https://user-images.githubusercontent.com/131881989/234584791-1a1b9f0b-e2f7-4a8c-9943-085ed161e62e.png)

**Describe the solution you'd like**
Use system settings.

**Describe alternatives you've considered**
Additional fields to explicitly specify these delimiters

**Additional context**
See like in excel: settings, special.

**Comment by LonwoLonwo** [comment]
Hello @gaborsodi 

You can change it with the Local setting (Language or Country):

![2023-04-26 16_55_46-Preferences](https://user-images.githubusercontent.com/45152336/234598845-a173ac12-4c15-49e1-a697-d5b9b0eb794f.png)
